src/packages/core/resources/resource.controller.ts

```diff
@@ -129,10 +141,11 @@ export class UmbResourceController extends UmbControllerBase {
 					default:
 						// Other errors
 						if (this.#notificationContext) {
+							const message = error.body?.errors ? this.#buildApiErrorMessage(error.body.errors) : undefined;
 							this.#notificationContext.peek('danger', {
 								data: {
 									headline: error.body?.title ?? error.name ?? 'Server Error',
-									message: error.body?.detail ?? error.message ?? 'Something went wrong',
+									message: message ?? error.body?.detail ?? error.message ?? 'Something went wrong',
```

Comment:
Would it make sense to forward the error messages in another property to the notification context so that you let that context handle the rendering and styling of the error messages? At least that way you could avoid defining `<div>`'s in the message property as you do on line 46?